crates/polars-core/src/series/arithmetic/borrowed.rs

```diff
@@ -171,6 +173,167 @@ impl NumOpsDispatchInner for FixedSizeListType {
     }
 }
 
+/// Given an ArrayRef with some primitive values, wrap it in list(s) until it
```

Comment:
Can we create a submodule `list_borrowed`? I think it is better to have this code local and this module a bit smaller.